### ISSUE TITLE
Encountering issue to pull private docker image from Dockerhub

### ISSUE THREAD

**Issue by martmull** [polyaxon/polyaxon#412]
### Unclear documentation
Hello,

Polyaxon is working fine when It pulls a public docker image stored on Dockerhub. 
However, it fails when it tries to pull it from a private repository on Dockerhub.

I followed [this documentation](https://github.com/polyaxon/polyaxon/blob/master/docs/configuration/private-registries.md) and I reinstalled Polyaxon adding the Dockerhub credentials in the `polyaxon_config.yaml` file such as:
```yaml
privateRegistries:
  - host: "https://registry-1.docker.io/v2/"
    user: "<USER_ID>"
    password: "<PASSWORD>"
```
but I can't make it work. It shows me this error:
![image](https://user-images.githubusercontent.com/29927851/55164677-61c30780-516c-11e9-8930-cbda39806411.png)

I also tried using this registry "https://index.docker.io/v1/", with and without the protocol, I always encounter the same issue. 

Do you have any idea why Polyaxon cannot access the private docker image? Or am I missing something?

Thank you very much,
and by the way, Polyaxon is awesome!

**Comment by mouradmourafiq** [comment]
I will check/reply with instruction later today or tomorrow, but I could not help not commenting on the `styles`. you can hard refresh your browser's tab to get latest js/css: `ctr+shift+r` or `cmd+shift+r`.

**Comment by martmull** [comment]
> I will check/reply with instruction later today or tomorrow, but I could not help not commenting on the `styles`. you can hard refresh your browser's tab to get latest js/css: `ctr+shift+r` or `cmd+shift+r`.

done

**Comment by martmull** [comment]
> I will check/reply with instruction later today or tomorrow
Thank you!

**Comment by mouradmourafiq** [comment]
@martmull this is a bug or rather a regression from the docker client. I was able to reproduce it, I also made a fix, it will be part of the next release v0.4.3 coming end of this week/early next week.

Thanks for reporting 👍 

N.B. the right way to reference docker hub registry is `https://index.docker.io/v1/`. I will probably make a separate integrations section for docker hub to make it clear for future references.

Closing the issue since a fix was already made.
